packages/simon/src/frontleft-classic-cuton.js

```diff
@@ -32,6 +32,12 @@ export default (part) => {
       .move(points.placketCfNeck)
       .line(points.placketCfHem)
       .attr('class', 'help')
+    if (!options.seperateButtonPlacket) {
```

Comment:
I'm not sure it makes sense to have this check. I am trying to avoid displaying a match line when there is a separate placket but it looks like this setting cannot be relied on for that. If I choose separate placket for either side of the shirt, but also set the corresponding placket style to seamless, the pattern does not actually contain a separate placket. Assuming this is a bug, maybe it makes more sense for me to address that issue first, then add this check. wdyt?

Reply:
The idea would be to prevent the user from choosing seamless when a separate placket has been selected.

Reply:
Yeah so the issue here is that *seamless* is a style without any seam line at the placket, and thus it is not possible to have a separate placket and seamless style. And in practice, the seamless setting takes precedent.

What's needed here is some way to designate options dependent on others and hide them in the frontend when they become irrelevant. So that when the user picks a 'seamless' placket style, the option to make it separate is no longer available.

That is not something we currently support, but it's on our roadmap now, and given that we're also building a new frontend, this is :100: something I want to add.

As to the matter at hand, we can leave the check in.

Reply:
Forgot the link to the roadmap: #1278